packages/client/components/ErrorBoundary.tsx

```diff
@@ -45,10 +46,11 @@ class ErrorBoundary extends Component<Props, State> {
         }
       })
       if (email) {
-        LogRocket.identify(atmosphere.viewerId, {
+        LogRocket.identify(viewerId, {
           email
         })
       }
+      window.localStorage.setItem(LocalStorageKey.ERROR_PRONE_AT, `${new Date().getTime()}`)
```

Comment:
itty bitty the absolute most nitty: Date.now() is a little cleaner over new Date().getTime()

Reply:
Thanks! I always appreciate the itty bitty nitty tips